_data/links.yml

```diff
@@ -5,3 +5,15 @@
 - text: Join our Discord!
   url: /discord
   image: /assets/images/discord-logo.png
+
+- text: "AFNOM's 'Missing Semester' signup form"
+  url: https://docs.google.com/forms/d/e/1FAIpQLSd0AqwJ774SfcQvd-j1bNQTeqIcBAjMEXFUro6hQbsmYYXIxw/viewform
```

Comment:
Could we replace this with `https://missingsemester.afnom.net/` please

Reply:
Fixed!